src/main/java/de/zalando/aruha/nakadi/validation/EventValidation.java

```diff
@@ -59,6 +61,14 @@ private static JSONObject wrapSchemaInData(final JSONObject schema, final EventT
         return wrapper;
     }
 
+    private static void moveDefinitionsToRoot(final JSONObject wrapper, final JSONObject schema) {
```

Comment:
I think we should have a test for the functionality that is performed in this method.